prime-router/metadata/HL7/catchall/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -295,6 +295,9 @@
 - id: "hl7v2-date-time"
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time"
 
+- id: "hl7v2Name"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Name"
```

Comment:
Removed the use of hl7v2Name here: https://github.com/CDCgov/prime-reportstream/pull/15910/commits/3764982f09d8651698a4df5642d688b9c398315c

This was a bit of a rabbit hole in two ways: 
1. The mapping file that was using this was mapping OBR.15.3 to `Specimen.note.text`. This would be correct, however it appears that HAPI is actually putting OBR.15.1 into `note.text`. Additionally, OBR.15.1 is _supposed_ to map  to `Specimen.type` however this is not happening. I suspect there are some datatype shenanigans going on here. The field type for OBR.15 is SPS in 2.5.1 and ST in 2.7.1. I will make a note to further investigate this in the followup ticket. 
2. Some mappings, notably ADT_A01 still rely on the hl7v2Name url for their mappings. I logged a ticket to fix this: https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/16077